wpilibNewCommands/src/test/java/edu/wpi/first/wpilibj2/command/CommandRequirementsTest.java

```diff
@@ -14,60 +14,54 @@
 class CommandRequirementsTest extends CommandTestBase {
   @Test
   void requirementInterruptTest() {
-    try (CommandScheduler scheduler = new CommandScheduler()) {
```

Comment:
I'm not sure how this works -- where is `scheduler` (used in the rest of the test) defined?